pootle/apps/pootle_statistics/models.py

```diff
@@ -551,6 +551,13 @@ def get_similarity(self):
     def is_similarity_taken_from_mt(self):
         return self.submission.similarity < self.submission.mt_similarity
 
+    def get_suggested_words(self):
```

Comment:
This sounds like it returns a list, can you name it `get_suggested_wordcount` or some such (and clarify the doc along with it).